### PR TITLE
Test Case: Update "returns" using enumerations #30 has been inplemented

### PR DESCRIPTION
**Pull request by AlekseyDidilov**
All possible reasons and conditions were declared within  'Reason', 'Condition' enum.
And this enum has to be used when a reason and condition needs to be set.

#30

### CODE REVIEW

**Comment by alexbeatnik** [comment]
Nice job!

File: oct/pages/returns.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
+from enum import Enum
 from selenium.webdriver import Remote
 from selenium.common.exceptions import NoSuchElementException
 from oct.pages.base import Page
 from oct.pages.registration import PersonalDetails
+from oct.pages.review import Rating as Reason
```
Comment (extsoft) [inline oct/pages/returns.py]: @AlekseyDidilov it is a bad idea to reuse the same enum for the different purpose even if values are the same.
What is the meaning of `Reason.ONE`? `ONE` has to be replaced with something meaningful.

File: oct/pages/returns.py [hunk 2]
```diff
@@ -1,7 +1,14 @@
+from enum import Enum
 from selenium.webdriver import Remote
 from selenium.common.exceptions import NoSuchElementException
 from oct.pages.base import Page
 from oct.pages.registration import PersonalDetails
+from oct.pages.review import Rating as Reason
+
+
+class Condition(Enum):
+    ZERO = 0
```
Comment (extsoft) [inline oct/pages/returns.py]: @AlekseyDidilov what is a meaning for `ZERO` condition? And for `ONE`?

File: oct/pages/returns.py (hunk 1 above)
Comment (AlekseyDidilov) [reply oct/pages/returns.py]: @extsoft Understood

Comment (AlekseyDidilov) [reply oct/pages/returns.py]: Class Reason has been created in oct/pages/returns.py. Ambiguous names has been replaced on meaningful.

File: oct/pages/returns.py (hunk 2 above)
Comment (AlekseyDidilov) [reply oct/pages/returns.py]: @extsoft Ambiguous names has been replaced on meaningful.

**Pull request merged**